src/library/styles/pxToEm.js

```diff
@@ -4,8 +4,20 @@
  */
 import fontSize_base from '../themes/fontSizeBase';
 
+const justNumbersOrEndsWithPx = (s: string) => s.match(/^[\d.]+(px)?$/);
+
+const convert = (input: number): string =>
+  `${input / parseInt(fontSize_base)}em`;
```

Comment:
This should prolly use `parseFloat` as well, no?

Reply:
`fontSize_base` will always be an integer value, as the entire system is built upon it, and we want easy math.